airflow/jobs.py

```diff
@@ -398,9 +398,9 @@ def schedule_dag(self, dag, session=None):
             for dr in active_runs:
                 if (
                         dr.start_date and dag.dagrun_timeout and
-                        dr.start_date < datetime.now() - dag.dagrun_timeout):
+                        dr.start_date < datetime.utcnow() - dag.dagrun_timeout):
```

Comment:
The idea is that we always use `datetime.utcnow()` to provide the start date. This way we can guarantee that the comparison would be as expected. The issue here is though that the default for the start_date is `func.now()` which will depend on the database timezone. Is it ok to use `datetime.utcnow()` instead over there? This way for airflow management purposes everything would be in utc regardless of system times.